src/commands/synthetics/renderer.ts

```diff
@@ -130,6 +130,23 @@ const renderResultOutcome = (result: Result, test: Test, icon: string, color: ty
   }
 }
 
+const renderApiRequestDescription = (subType: string, request: Test['config']['request']): string => {
+  if (subType === 'dns' && request.host) {
```

Comment:
`request.host` is expected to be always defined?
Otherwise, at first glance, it seems if it's not, the `dns` request will be displayed like the default which looks more like an `http` than a default.

Reply:
🤦 my apologies, it is always defined for DNS test results, I forgot to remove this check when refactoring my code.